finance.reckoner.app.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: finance.reckoner.app
```

Comment:
`app` isn't an application name.

Reply:
This is to match the URL for the web app. This is just the app ID, not the name.

Reply:
> The format is in reverse-DNS style so the first section should be a domain controlled by the project and the trailing section represents the specific project

So the last componet IS the application name. 
Also verification will be done against `reckoner.finance` in that case.
`app` is a bad name. `Reckoner` is likely appropriate.